frontend/src/AlertConfiguration/AlertConfigurationComponents/ConfigForm.js

```diff
@@ -0,0 +1,49 @@
+import React, { useState, Fragment } from 'react';
```

Comment:
hmm, this `ConfigForm.js` file seems to look the same as the`ConfigCard.js` file...

was wondering about the reason for the duplication?